pythonPackages.nix

```diff
@@ -53,6 +53,11 @@ let
 
     rmsd = callPackage ./pkgs/lib/rmsd { };
 
+    tblite = callPackage ./pkgs/lib/tblite/python.nix {
+      inherit (selfPkgs) tblite;
+      inherit (superPkgs) meson;
```

Comment:
Does it pick up a python package with the same name if you remove that line?

Reply:
Exactly, name [python3Packages.meson](https://search.nixos.org/packages?channel=22.11&from=0&size=50&sort=relevance&type=packages&query=python+meson)